package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "MVP-Test-Harness",
+  "name": "Rosie Prototype",
```

Comment:
Both instances of `name` here will need to be the same as the `name` in package.json. Running an npm install will ensure everything is in sync.

Reply:
Ok, I will fix that